src/backends/glvisualize.jl

```diff
@@ -826,7 +826,7 @@ function gl_bar(d, kw_args)
     hw = if bw == nothing
         mean(diff(x))
     else
-        Float64[cycle(bw,i)*0.5 for i=1:length(x)]
+        Float64[_cycle(bw,i)*0.5 for i=1:length(x)]
```

Comment:
I'll cherry pick for the cycle change and the `@userplot` change, and close this as a messy pre-PR.